marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/data-hub-security-admin.json

```diff
@@ -35,6 +35,11 @@
       "action": "http://marklogic.com/xdmp/privileges/remove-amp",
       "kind":"execute"
     },
+    {
+      "privilege-name": "amp-set-roles",
```

Comment:
Is this privilege only needed on 10.0-4.4 or higher in order for updating an amp to work? If so, add a message to one of the assertions below to state that.